VimeoNetworking/Sources/VimeoSessionManager+Constructors.swift

```diff
@@ -33,31 +33,33 @@ public extension VimeoSessionManager
     /**
      Creates an authenticated session manager with a static access token
      
+     - parameter baseUrl: the base URL for the HTTP client
      - parameter accessToken: the static access token to use for request serialization
      
      - returns: an initialized `VimeoSessionManager`
      */
-    static func defaultSessionManager(accessToken: String) -> VimeoSessionManager
+    static func defaultSessionManager(baseUrl: URL = VimeoBaseURL, accessToken: String) -> VimeoSessionManager
```

Comment:
Should we be specifying a default in multiple places? I'm worried someone using a custom base URL might use the default here by mistake, then have a session manager with a different URL than the client or something. Might be worth requiring them to pass it in. Same with below.